src/main/java/frc/robot/autonomous/AutonomousConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.autonomous;
+
+import com.pathplanner.lib.path.PathConstraints;
+
+
+public class AutonomousConstants {
+
+	public static final String LOG_PATH_PREFIX = "Autonomous/";
+
+	public static final PathConstraints REAL_TIME_CONSTRAINTS = new PathConstraints(2.5, 2.5, 4, 4);
+
+	public static final double CLOSE_TO_TARGET_POSITION_DEADBAND_METERS = 0.5;
```

Comment:
What is this?

Reply:
A constant

Reply:
It was in swerve but it is more an auto constant than a swerve constant

Reply:
Yeah but why do you need this where do you use this what does it mean

Reply:
in swerve commands builder if you want to understand more, I'll explain in הכשרת אוטונומים